Sources/Mistica/Components/Header/HeaderView.swift

```diff
@@ -225,42 +208,40 @@ private extension HeaderView {
         setContentHuggingPriority(.required, for: .vertical)
         setContentCompressionResistancePriority(.required, for: .vertical)
 
-        updateLayoutMarginsGuide()
+        updateColors()
 
         addSubview(stackView, constraints: [
-            stackView.leadingAnchor.constraint(equalTo: leadingAnchor),
-            stackView.topAnchor.constraint(equalTo: topAnchor),
-            stackView.trailingAnchor.constraint(equalTo: trailingAnchor),
-            bottomAnchor.constraint(equalTo: stackView.bottomAnchor)
+            stackView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: Constants.marginLeft),
+            stackView.topAnchor.constraint(equalTo: topAnchor, constant: Constants.marginTop),
+            stackView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -Constants.marginRight),
```

Comment:
To avoid using negative constraints I think it's enough changing the order:

```suggestion
            trailingAnchor.constraint(equalTo: stackView.trailingAnchor, constant: Constants.marginRight),
```

Reply:
done!